packages/mds-geography-author/api.ts

```diff
@@ -24,14 +24,10 @@ function api(app: express.Express): express.Express {
     }),
     async (req, res) => {
       const { scopes } = res.locals
-      const { get_published = null, get_unpublished = null } = req.query
-      const params = { get_published, get_unpublished }
-      if (get_published) {
-        params.get_published = get_published === 'true'
-      }
-
-      if (get_unpublished) {
-        params.get_unpublished = get_unpublished === 'true'
+      const { get_published, get_unpublished } = req.query
+      const params = {
+        get_published: get_published ? get_published === 'true' : null,
+        get_unpublished: get_unpublished ? get_unpublished === 'true' : null
```

Comment:
No fancy JSON.parse based parser for booleans here? I see some elsewhere. Might make sense to have some standard parsers, like `parseQueryNumbers`, `parseQueryBoolean`, etc. that can pass in some standard parsers so folks don't have to keep writing them.

Reply:
Agreed -- gonna hold off standardizing those parsers until another PR though I think.